XRTK-Core/Packages/com.xrtk.core/Editor/PackageInstaller.cs

```diff
@@ -133,6 +133,12 @@ private static void AddConfigurations(List<string> profiles)
         {
             AssetDatabase.Refresh(ImportAssetOptions.ForceUpdate);
 
+            try
+            {
+                Selection.activeGameObject = null;
```

Comment:
I'd like to see this be `Selection.activeObject` in case we have a profile selected.

Reply:
Fixed